docs/source/index.rst

```diff
@@ -28,11 +28,17 @@ Features described in this documentation are classified by release status:
   binary distributions like PyPI or Conda, except sometimes behind run-time
   flags, and are at an early stage for feedback and testing.
 
+.. toctree::
+   :glob:
+   :maxdepth: 1
+   :caption: Community
+
+   community/*
 
 .. toctree::
    :glob:
    :maxdepth: 1
-   :caption: Notes
+   :caption: Guides
```

Comment:
What is the reason for this rename? This section is always refered as the "Notes" section. So there might be other places we need to change if we want to rename this.

Reply:
Good callout. Given that tech governance will have things like meeting notes, etc. it seemed like Notes here would eventually be confusing. Any thoughts on whether this is the right title or not? Also where else this would need to change?

Reply:
> Also where else this would need to change?

I know there are forum discussions or github issues where we reference to the "Note in the doc about Extending PyTorch". Not sure we need to change these.
A couple of the entries in there do start with "In this note we’ll cover ..." which we might want to change?
We might want to rename the folder that contain the sources though to avoid confusion for people looking to modify these.

> Any thoughts on whether this is the right title or not?

I think these are really `Developer Notes` would that be a better name?

Reply:
I think `Developer Notes` works for me. @svekars would you mind putting up a separate PR with that change? I think you're making some other edits as well, so shouldn't be too painful. Thank you!

Reply:
Adding in https://github.com/pytorch/pytorch/pull/78552 along with the other change.